pkg/kudoctl/cmd/operator.go

```diff
@@ -0,0 +1,36 @@
+package cmd
+
+import (
+	"io"
+
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+const operatorDesc = `
+This command consists of multiple sub-commands to interact with KUDO operators.
+
+It can be used to package or verify an operator, or list parameters.  When working with parameters it can 
+provide a list of parameters from a remote operator given a url or repository along with the name and version.
+`
+
+const operatorExamples = `  kubectl kudo operator package [operator folder]
```

Comment:
wondering why did you go with the extra operator level here. To me it's kind of redundant... Do you think that following would not be enough?

`kudo package params`
`kudo package verify`
`kudo package info`

or something along those lines

Reply:
the use of `package` is challenging if we use `package` to package an operator.   It works if we perhaps add a `package create`.... which is currently `operator package`

Reply:
prior to this PR... `kudo package` was an action... it took an operator folder and made a package.   This PR and feature is provide info back... it seems slightly incongruent... but the issue is `package [folder]` and `package params [xyz]` doesn't work together.   If we are stuck on putting it all under package... than we would need a `package create [folder]` for the current `package` feature.